pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -66,6 +71,8 @@ class MCPServer(AbstractToolset[Any], ABC):
     allow_sampling: bool = True
     max_retries: int = 1
     sampling_model: models.Model | None = None
+    allow_elicitation: bool = True
+    elicitation_callback: ElicitationFnT | None = None
```

Comment:
We should document this in `docs/mcp/client.md` -- it can just be an Elicitation section that links to the official docs and has an example of setting the callback.